common/install-tools.sh

```diff
@@ -26,6 +26,15 @@ echo "### "
 echo "### Begin Tools install"
 echo "### "
 
+## Install tree
+## Note: This is here and not in install-tools.sh to ensure it is available across sessions since this is installed using apt-get
```

Comment:
This won't be available across sessions now.

Reply:
good point, let me see if I can move the binary to the workdir.